### PR TITLE
feat: list

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot (93)
Screenshot (94)

### CODE REVIEW

File: Components/list.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  style="width: 100%; height: 150px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
```
Comment (Aditi0505) [inline Components/list.html]: Can fix indentation here.

**Review by Aditi0505 — APPROVED** [review]
Looks good to me!

**Review by anupkgurung — APPROVED** [review]
Please format the code before pushing. Rest looks good.

**Comment by aniketsingh13** [comment]
> Please format the code before pushing. Rest looks good.

will do that

File: Components/list.html (hunk 1 above)
Comment (aniketsingh13) [reply Components/list.html]: will do that

**Pull request merged**